### PR TITLE
Add Securing with JWS

### LINKED ISSUE #201: Support JWT-secured VCs
The current specification does not support JWT-secured VCs and focuses on SD-JWT instead. This is concerning for two reasons:

1. The WG's charter expires in six months and it is doubtful that SD-JWT will reach IETF RFC status in that timeframe, which will leave the specification with no fallback for JOSE-based mechanisms.
2. If the goal of the specification is "to be implemented with standards for signing and encryption that are widely adopted." (as the Abstract states), then SD-JWT does not meet that criteria, and there does not seem to be broad support for COSE-secured VCs in any community at present.

At a minimum, the specification should elaborate on how a VC can be secured using a JWT. 

Section 3.1.1 adds to the confusion by referencing RFC7515 (JSON Web Signatures), but then specifies the media type as `vc+ld+json+sd-jwt`. This is confusing because RFC7515 does not contain any information on how to secure anything using SD-JWT.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #201

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/vc-jose-cose/pull/239.html" title="Last updated on Feb 20, 2024, 6:11 PM UTC (8ea79f5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/239/6874bf8...selfissued:8ea79f5.html" title="Last updated on Feb 20, 2024, 6:11 PM UTC (8ea79f5)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,38 +153,43 @@
...
     <p>
-      Selective Disclosure for JWTs (SD-JWT) [[SD-JWT]] provides a standardized
-      mechanism for digitally signing JSON documents. It provides a means
-      to ensure the integrity, authenticity, selective disclosure and non-repudiation of
-      the information contained in a JSON document. These
-      properties make SD-JWT especially well suited to securing documents
-      conforming to the JSON-LD [[VC-DATA-MODEL-2.0]].
+      JSON Web Signature (JWS) [[RFC7515]] defines a standard means of
+      digitally signing documents, including JSON documents,
+      jsing JSON-based data structures.
```
Comment (OR13) [inline index.html:171]: Jsing?

**Review by OR13 — APPROVED** [review]

**Comment by iherman** [comment]
I think this was already mentioned elsewhere (@decentralgabe may remember): in [Example 26](https://pr-preview.s3.amazonaws.com/selfissued/vc-jose-cose/pull/239.html#example-presentation) the list for `verifiableCredential` is erroneous. "Pure" URL-s may not appear there...

File: index.html [hunk 2]
```diff
@@ -237,8 +244,11 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          [[RFC7515]] MAY be used to secure this media type.
-          The <code>typ</code> header parameter SHOULD be <code>vc+ld+json+sd-jwt</code>.
+          JWS [[RFC7515]] is used to secure this media type with JOSE.
```
Comment (decentralgabe) [inline index.html:247]: should we add a `MUST` here?

File: index.html [hunk 3]
```diff
@@ -283,7 +293,114 @@ <h2>Securing JSON-LD Verifiable Presentations with JOSE</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          [[RFC7515]] MAY be used to secure this media type.
+          [[RFC7515]] is used to secure this media type in this case.
```
Comment (decentralgabe) [inline index.html:296]: `MUST` here as well?

**Comment by decentralgabe** [comment]
@iherman I see what you're saying, there is an issue with our example rendering. @OR13 has graciously taken on some work to fix this, for which I have opened #240 to track this and referenced your issue.

I would ask that we view Mike's change set independent of the example rendering issues, which I will work to fix in the coming days.

**Review by decentralgabe — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (selfissued) [reply index.html:247]: No need.  It means the same thing with and without the MUST.  Plain English text is just as normative as RFC 2119 language.  And it reads more naturally as is.

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -152,38 +153,43 @@
...
+      It offers a flexible and extensible set of
       cryptographic options, allowing for a wide range of algorithms
       to be used for signing and encryption.
-
+    </p>
+    <p>
       COSE supports two main operations: signing and encryption. For
       signing, COSE allows the creation of digital signatures over
-      CBOR data using various algorithms such as HMAC, RSA, ECDSA, or
+      CBOR data using various algorithms such as RSA, ECDSA, and
```
Comment (TallTed) [inline index.html:192]:
```suggestion
CBOR data using various algorithms including RSA, ECDSA, and
```

**Pull request merged**